Makefile

```
@@ -1,2 +1,5 @@
 build:
-	g++ src/*.cpp -std=c++17 -lzip -lcurl -o tldr
\ No newline at end of file
+	g++ src/*.cpp -std=c++17 -lzip -lcurl -o tldr
+
+install: 
+	cp ./tldr /usr/loca/bin/
```

```suggestion
	cp ./tldr /usr/local/bin/
```
Why not `/usr/bin/`? I don't really get the difference.

---

I don't see the problem with /usr/bin, if you prefer it, I will change to it.